Common/src/geometry/CGeometry.cpp

```diff
@@ -3108,7 +3108,7 @@ void CGeometry::ComputeSurf_Curvature(CConfig *config) {
 }
 
 void CGeometry::FilterValuesAtElementCG(const vector<su2double> &filter_radius,
-                                        const vector<pair<unsigned short,su2double> > &kernels,
+                                        const vector<pair<ENUM_FILTER_KERNEL,su2double> > &kernels,
```

Comment:
enum class is the safe way to remove the numbers... it breaks the code everywhere :)